src/jupyterlabcontroller/exceptions.py

```diff
@@ -519,9 +519,26 @@ def to_slack(self) -> SlackMessage:
 
 
 class MissingSecretError(MissingObjectError):
-    """Secret specified in the controller configuration was not found."""
+    """Secret specified in the controller configuration was not found.
 
-    def __init__(self, message: str, namespace: str, name: str) -> None:
+    Parameters
+    ----------
+    name
+        Name of secret.
+    namespace
+        Namespace of secret.
+    key
+        Key within that secret that was found if the secret was present but
```

Comment:
I find the wording of the comment very confusing, although what it's doing is quite clear from the code.  Maybe when the coffee kicks in I will have a constructive solution.